app/lib/url.rb

```diff
@@ -56,6 +56,14 @@ def self.user(user)
     nil
   end
 
+  # Creates an Image URL - a shortcut for the .image_url helper
+  #
+  # @param image_name [String] the image file name
+  def self.local_image(image_name)
+    host = ActionController::Base.asset_host ||= url(nil)
+    ActionController::Base.helpers.image_url(image_name, host: host)
+  end
```

Comment:
I did some more digging on this and it turns out unless you explicitly define `asset_host`, `image_url` will basically return a relative path.

In the DEV instance, we set the `asset_host` https://github.com/forem/forem/blob/034b049e3d69ce2417440decd4c09bcf91f623f4/config/environments/production.rb#L54

In other instances, we do _not_ define `FASTLY_CDN_URL` - making it `nil` and returning a `nil` host. When you provide a `host` to `image_url`, it overrides whatever is set as `assset_host`. So here, we basically reassign the `host` explicitly if one is defined (i.e. on the DEV instance where we use the Fastly CDN to serve assets) or we set it to the app protocol/domain (i.e. every other instance of Forem right now) to return a pretty URL.

![Screen Shot 2020-10-28 at 3 30 07 PM](https://user-images.githubusercontent.com/15987080/97487354-25045700-1933-11eb-862f-1c846a425451.png)

Another option is to change the logic in `production.rb` so that it sets the `asset_host` based on this same criteria. But I figured we can re-visit that for when we move other instances of Forem to a CDN and we have to set that value at that point anyways. What do you think?

cc: @benhalpern @vaidehijoshi @citizen428 @jdoss